e2e/testdata/fn-eval/function-chain/.expected/exec.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+kpt fn source |\
+kpt fn eval - --image gcr.io/kpt-fn/set-namespace:v0.1 -- namespace=staging |\
+kpt fn eval - --image gcr.io/kpt-fn/set-label:v0.1 -- foo=bar |\
+kpt fn sink .
```

Comment:
do you need `.`? It should default to current directory.

Reply:
We need `.`. For sink, the default behavior is printing to stdout if there is no directory given.

Reply:
`DIR` is required parameter for `sink`.

Reply:
Why? This is inconsistent with other commands. Let's follow up on this. @phanimarupaka